lib/src/pubsub_impl.dart

```diff
@@ -428,7 +429,10 @@ class _SubscriptionImpl implements Subscription {
   Future delete() => _api._deleteSubscription(_subscription.name!);
 
   @override
-  Future<PullEvent?> pull({bool wait = true}) {
+  Future<PullEvent?> pull({
+    @Deprecated('returnImmediately has been deprecated from pubsub')
```

Comment:
Perhaps leave a link to the documentation...

Reply:
there isn't really any.

Reply:
Well you could have linked to: https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.subscriptions/pull

Reply:
never mind...